environment.yml

```
@@ -0,0 +1,35 @@
+#name: z
+channels:
+  - defaults
+  - ome
+  - conda-forge
+  - anaconda
```

`anaconda` is a weird metachannel, which isn't really intended for public use. `conda-forge` should be higher priority than `defaults` as that is how `conda-forge` packages are built and intended to be used.

```suggestion
channels:
  - ome
  - conda-forge
  - defaults
```